tests/test_vcfline.py

```diff
@@ -182,14 +181,44 @@ def test_combine_format_values_by_sample_as_str(self):
         # Now prints all 3 samples
         assert vcf_line1.combine_format_values_by_sample_as_str()  == '0/1:5:25:50,0,12\t1/1:15:50:120,45,0\t0/0:2:2:0,10,10'
 
+    def test_fill_merge_dicts(self):
+        merged_info_dict = {'END': '128', 'AD': '3', 'VARSEQ': '.', 'ALIAS': 'CNV5711', 'ID': '14', 'VARCALLSOID': 'SO:0001742', 'SVCID': 'CNV5711', 'DBXREF': 'mydata', 'NAME': 'nssv1388955', 'AC': '3', 'SVLEN': '1'}
+        key = "VARCALLSOID"
+        previous_line_info_value = "SO:0001742"
+        current_line_info_value = "SO:1234567"
+        merged_info_dict= VcfLine.fill_merge_dicts(self, merged_info_dict, key, previous_line_info_value, current_line_info_value)
+        expected_merged_info_dict = {'END': '128', 'AD': '3', 'VARSEQ': '.', 'ALIAS': 'CNV5711', 'ID': '14', 'VARCALLSOID': 'SO:0001742,SO:1234567', 'SVCID': 'CNV5711', 'DBXREF': 'mydata', 'NAME': 'nssv1388955', 'AC': '3', 'SVLEN': '1'}
+        assert merged_info_dict == expected_merged_info_dict
+
     def test_merge_info_dicts(self):
-        pass
+        VcfLine.merge_info_dicts(
+            self.vcf_line, self.other_vcf_line
+        )
+        expected_merge_info_dict = {'NAME': 'nssv1412199,nssv9912199', 'SVLEN': '1,4'}
+        assert self.vcf_line.info_dict == expected_merge_info_dict
+        assert self.other_vcf_line.info_dict == expected_merge_info_dict
+
+    def test_merge_vcf_values_for_format(self):
+        VcfLine.merge_vcf_values_for_format(self.vcf_line, self.other_vcf_line)
+        expected_merge_vcf_values_for_format = {'sample1': {'GT': '0/1'}, 'sample2': {'GT': '0/1'}}
+        assert self.vcf_line.vcf_values_for_format == expected_merge_vcf_values_for_format
 
-    def test_merge_info_string(self):
-        pass
+    def test_format_info_string(self):
+        formatted_info_string = VcfLine.format_info_string(self.vcf_line)
+        expected_info_string = "NAME=nssv1412199;SVLEN=1"
+        assert formatted_info_string == expected_info_string
 
     def test_merge(self):
-        pass
+        list_of_samples = ['sample1']
+        VcfLine.merge(self.vcf_line, self.other_vcf_line, list_of_samples)
+        assert self.vcf_line.id == self.other_vcf_line.id
+        assert self.vcf_line.alt == self.other_vcf_line.alt
+        assert self.vcf_line.filter == self.other_vcf_line.filter
+        assert self.vcf_line.info_dict == self.other_vcf_line.info_dict
```

Comment:
I don't think that we expect the two vcf to be the same. 
I expect the `vcf_line` to be the union of `vcf_line` and `other_vcf_line`